electron-builder.yml

```diff
@@ -50,6 +50,12 @@ fileAssociations:
     ext: prototxt
   - name: "Caffe Model"
     ext: prototxt
+  - name: "OpenVINO IR Model"
```

Comment:
`xml`, `bin` and `dot` are rather general extensions reserved for other tools. They should not be registered to this app. Please remove.

Reply:
Do you have some context why there are 3 new formats needed. Can't OpenVINO support ONNX or one of the other existing formats instead of introducing multiple new ones?

Reply:
Either way the change in `electron-builder.yml` should be reverted (same as `json` for example). Those extensions are too general and more likely than not `xml`, `bin` and `dot` files are not OpenVINO files and should not be opened in Netron.

Reply:
@lutzroeder could you please advice how to properly register those extensions, where is the `json` example?

Reply:
They are not registered and user will have to open the app and drag & drop or use the file dialog. This isn't ideal but there has been pushback from users and partners to not register general extensions. Even `pb` is at risk of getting removed.

Reply:
This will get removed -- right?

Reply:
@lutzroeder I removed the usage.

is the discussion resolved now?